src/test/compile-fail/coercion-slice.rs

```diff
@@ -14,6 +14,5 @@ fn main() {
     let _: &[i32] = [0];
     //~^ ERROR mismatched types
     //~| expected type `&[i32]`
-    //~| found type `[{integer}; 1]`
```

Comment:
I have absolutely no clue from where this change comes since my code just added suggestion.